server/schema.graphql

```diff
@@ -0,0 +1,141 @@
+enum AnimationObjectTypes {
+  triangle
+  offence
+  defence
+  disc
+}
+
+type Animation {
+  # all three fields should have same length
+  ids: [AnimationObjectTypes!]!
+  texts: [String]
+  positions: [# ids length
+  [# steps length
+  [# 2 items - cut and final
+  [# 2 items - X and Y
+  Float]]]]
+}
+
+input InputAnimation {
+  # all three fields should have same length
+  ids: [AnimationObjectTypes!]!
+  texts: [String]
+  positions: [# ids length
+  [# steps length
+  [# 2 items - cut and final
+  [# 2 items - X and Y
+  Float]]]]
+}
+
+type DrillStep {
+  title: String!
+  subtitle: String
+}
+
+enum DrillTypes {
+  FITNESS
+  TECHNICAL
+}
+
+enum Intensities {
+  LOW
+  MODERATE
+  HIGH
+}
+
+enum GoalsFrisbee {
+  DEFENSE
```

Comment:
Careful with using this kind of enums. Because each key corresponds to a numeric value things start to get very messy as soon as you either remove or add new entries. Personally I prefer to use [string enums](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums) as they are a lot easier to maintain in the long run